Libraries/LibLine/Editor.cpp

```diff
@@ -126,6 +126,36 @@ static size_t codepoint_length_in_utf8(u32 codepoint)
     return 3;
 }
 
+Editor::CodepointRange Editor::scan_for_byte_offsets(size_t start_offset, size_t end_offset, size_t scan_offset, bool reverse) const
+{
+    size_t byte_offset = 0;
+    size_t codepoint_offset = scan_offset + reverse;
```

Comment:
Adding a `bool` to a `size_t`? If that is what you intended to do, maybe write it explicitly?

Reply:
Fine, I'll cast it.
those C habits die hard

Reply:
I mean maybe `reverse ? scan_offset + 1 : scan_offset`

Reply:
but...why introduce a conditional branch that the compiler might decide not to optimise away?

Reply:
Let's always prioritize clarity until profiling proves that something is a bottleneck :)

Reply:
Certainly, but `number + boolean` was very clear to me.
regardless, I changed it to be explicit `foo + (bar ? 1 : 0)`